src/RtAudioInterface.cpp

```diff
@@ -69,16 +77,22 @@ RtAudioInterface::~RtAudioInterface()
     if (mRtAudio != NULL) {
         delete mRtAudio;
     }
+
+    if (mStereoToMonoMixer != NULL) {
+        delete mStereoToMonoMixer;
+    }
 }
 
 //*******************************************************************************
 void RtAudioInterface::setup(bool verbose)
 {
     // Initialize Buffer array to read and write audio and members
-    mNumInChans  = getNumInputChannels();
-    mNumOutChans = getNumOutputChannels();
-    mInBuffer.resize(getNumInputChannels());
-    mOutBuffer.resize(getNumOutputChannels());
+    mNumInChans  = AudioInterface::getNumInputChannels();
```

Comment:
This looks recursive.

Reply:
Argh. It is. That probably happened because RtAudioInterface declares mNumInChans and mNumOutChans a second time. Should be removed.